### PR TITLE
Update Intermagnet structure

### PR DESCRIPTION
**Pull request by SimonFlower**

### CODE REVIEW

**Review by ALewis-GA — APPROVED** [review]
Thanks Simon,

Looks good.  

Will you add your Sopron Report to IMOs into the meetings page or shall I  do that?

**Comment by SimonFlower** [comment]
Thanks Andrew. I've added my report to IMOs into this update as well. 

If Juergen is happy, I'll merge the changes and update the website.

File: structure.md
Comment (juergen-matzka) [inline structure.md:1]: after line 11 we should include:
- Andrew Lewis (Australia)
- and after line 12
- Jürgen Matzka (Germany)

File: contacts.md
Comment (juergen-matzka) [inline contacts.md:1]: after line 13 we should include 
| Andrew Lewis | andrew.lewis@ga.gov.au | +61 2 6249 9764 | Australia |
after line 14 we should include 
| Jurgen Matzka | jmat@gfz-potsdam.de | +49 (0)331-6264-1232 | Germany |

**Review by juergen-matzka — CHANGES_REQUESTED** [review]
Hi Simon and Andrew,

I made two comments to add Andrew and me to the ExCon list in contacts and structure

Best wishes

**Comment by SimonFlower** [comment]
Thanks Juergen - I've made those changes (and hopefully spelled your name correctly as well) - is that all OK now?

**Review by juergen-matzka — APPROVED** [review]
OK, Thanks!

**Pull request merged**